google-cloud-pubsub/lib/google/cloud/pubsub.rb

```diff
@@ -371,9 +382,10 @@ module Cloud
     #
     # subscriber = sub.listen do |msg|
     #   # process msg
-    #   msg.ack!
+    #   msg.acknowledge!
```

Comment:
Optional Nit: I'm a lazy, and appreciated seeing `msg.ack!` is available as well.

Reply:
For consistency, we decided to change `ack!` (which is an alias) to `acknowledge!` everywhere.

Reply:
FWIW, I agree I would rather see `ack!` and `nack!` in these examples. However, we just found out the doc site doesn't list it, so it might be more prudent to not show the aliases in the code examples.

Reply:
It seems we need to create a gcloud-common issue for listing aliases.

Reply:
Ah okay, could you file an issue to track this? 

Understood, I'd rather use what's being documented as to not cause friction when searching the docs for `ack!` versus `acknowledge!`. Keep `acknowledge!`.